cmake/External/nccl.cmake

```diff
@@ -20,6 +20,13 @@ if (NOT __NCCL_INCLUDED)
     set(NCCL_CXX_FLAGS ${CMAKE_CXX_FLAGS} ${NCCL_EXTRA_COMPILER_FLAGS})
     set(NCCL_C_FLAGS ${CMAKE_C_FLAGS} ${NCCL_EXTRA_COMPILER_FLAGS})
 
+    if (${TORCH_CUDA_ARCH_LIST})
+      torch_cuda_get_nvcc_gencode_flag(NVCC_GENCODE)
+      string(REPLACE "-gencode;" "-gencode=" NVCC_GENCODE "${NVCC_GENCODE}")
+    endif()
+
+    message("LOGGING GENCODE: ${NVCC_GENCODE}")
```

Comment:
will remove this line; i just don't want to kick off a whole new CI cycle right now